bin/commands/components/enable/.examples

```diff
@@ -0,0 +1,4 @@
+    zwe components enable --component-name gateway
+
+    zwe components enable --component-name gateway --ha-instance lpar1
+    
```

Comment:
A small change on the recent v2.x/staging will automatically indent the zwe .examples content. So no need to add 4 spaces in front of each line now.